cmd/dexctl/command_config.go

```diff
@@ -51,7 +52,7 @@ func runSetConnectorConfigs(cmd *cobra.Command, args []string) int {
 		r = f
 	}
 
-	cfgs, err := connector.ReadConfigs(r)
+	cfgs, err := readConfigs(r)
```

Comment:
It returns []ConnectorConfig and the admin api requires []interface{} in set connector method.
I wanted to keep clean the new admin api based implementation from haveing code converting ConnectorConfig to interface{}